src/tasks/generateSignatures.ts

```diff
@@ -16,6 +17,12 @@ const generateSignatures: () => void = () => {
       "fileWithAddress",
       "File with comma separated addresses that should be allow-listed",
     )
+    .addOptionalParam(
+      "postToApi",
+      "File with comma separated addresses that should be allow-listed",
+      "false",
+      types.boolean,
+    )
```

Comment:
You could use a flag, so you can call it with `--post-to-api` instead of `--post-to-api true`

```suggestion
    .addFlag(
      "postToApi",
      "File with comma separated addresses that should be allow-listed",
    )
```